bedrock/firefox/templates/firefox/browsers/mobile/index.html

```diff
@@ -112,7 +112,7 @@ <h3 class="mzp-c-menu-list-title">{{ ftl('browsers-mobile-download') }}</h3>
     </div>
 
     <div class="c-landing-grid-item">
-      <img src="{{ static('img/firefox/browsers/mobile/index/compare-mobile-browsers.svg') }}" alt="" class="c-landing-grid-img">
+      <img src="{{ static('img/firefox/browsers/mobile/index/compare-mobile-browsers.svg') }}" alt="" class="c-landing-grid-img" width="350" height=""350>
```

Comment:
Missed aim on quotes:

```suggestion
      <img src="{{ static('img/firefox/browsers/mobile/index/compare-mobile-browsers.svg') }}" alt="" class="c-landing-grid-img" width="350" height="350">
```